SpatialGDK/Source/SpatialGDK/Private/SpatialView/EntityPresenceRecord.cpp

```diff
@@ -0,0 +1,38 @@
+﻿#include "SpatialView/EntityPresenceRecord.h"
+
+namespace SpatialGDK
+{
+
+void EntityPresenceRecord::AddEntity(Worker_EntityId EntityId)
+{
+	if (!EntitiesRemoved.RemoveSwap(EntityId))
+	{
+		EntitiesAdded.Add(EntityId);
+	}
+}
+
+void EntityPresenceRecord::RemoveEntity(Worker_EntityId EntityId)
+{
+	if (!EntitiesAdded.RemoveSingleSwap(EntityId))
```

Comment:
Could you explain why this uses removesingleswap, but the func above uses just removeswap?

Reply:
Above is a mistake, will fix.